src/emailUnpackFiles/sendAutoEmail.js

```diff
@@ -0,0 +1,55 @@
+// function to send email to student for low grade
+/*
+data: JSON.stringify(data),
+contentType: 'application/json'
+*/
+import axios from 'axios';
+
+const inputs = {
+  class: 'Math',
+  studentName: 'michael',
+  studentEmail: 'mmmcclan365@gmail.com',
+  gradeThreshold: '70',
+  teacherEmail: 'mmmcclan365@gmail.com',
+  message: 'You are failing!',
+};
+// needs to take in inputs
+async function autoEmail() {
+  const keys = {
+    serviceid: 'service_3kyf5ip',
+    publickey: 'lANtIwYBJdwYhxvdz',
+    templateKey: 'template_dbshsoq',
+  };
+  const data = {
```

Comment:
are these keys going to live in .env?

If so we should add the variables to .env then import dotenv into this file so we don't have to refactor later and can just update the .env variables.

Also, if you do add them to .env then lets also add them to the sample.env file as well though we can leave the values blank. This way, when deploying a new instance of the app its clear what .env variables need to be set.